doc/api/os.md

```diff
@@ -1315,8 +1323,9 @@ The following process scheduling constants are exported by
   </tr>
 </table>
 
+[Android building]: https://github.com/nodejs/node/blob/master/BUILDING.md#androidandroid-based-devices-eg-firefox-os
```

Comment:
@Trott  Just re-order and put a new reference!

Reply:
I always call it alphabetical order, but it's actually ASCII order so that it can be easily correctly ordered by a tool. So the original location for this was correct. And your EUID link would come after it. Sorry for all the confusing particulars here!